.github/workflows/github-actions-native-image-containerisation.yml

```diff
@@ -8,23 +8,35 @@ permissions:
   contents: read
 jobs:
   build:
-    name: Native Image Containerisation on ${{ matrix.os }}
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [macos-latest, ubuntu-latest]
+    name: Native Image Containerisation
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
 
       - uses: graalvm/setup-graalvm@v1
         with:
-          version: 'latest'
-          java-version: '17'
-          components: 'native-image'
+          java-version: '24'
+          distribution: 'graalvm'
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Build and Test Java Code
         run: |
           cd native-image/containerisation/lab/
           ./mvnw --no-transfer-progress clean package
           ./mvnw --no-transfer-progress package -Pnative
+
+          # I. Build and run a JAR in a container
+          docker build -f ./00-containerise/Dockerfile \
+                --build-arg JAR_FILE=./target/jibber-0.0.1-SNAPSHOT.jar \
+                -t jibber:jdk.01 .
```

Comment:
I removed `docker run` checks deliberately as `curl` command fails to connect to the server.